projects/ontimize-web-ngx/src/lib/components/table/extensions/footer/paginator/o-base-table-paginator.class.ts

```diff
@@ -28,6 +28,18 @@ export class OBaseTablePaginator implements OTablePaginator {
     this._pageIndex = value;
   }
 
+  get pageSizeOptions(): Array<any> {
+    return this._pageSizeOptions;
+  }
+
+  set pageSizeOptions(value: Array<any>) {
+    if (typeof value == 'string') {
+      this._pageSizeOptions = JSON.parse(value)
```

Comment:
Protect this code, if the parse fails it would be great to assign default values